src/validators/validateAndroidStyle.ts

```diff
@@ -19,9 +22,12 @@ import { checkForProperty, isArray, isBoolean, isNumber, isObject, isString } fr
 export function validateAndroidBigPictureStyle(
   style: AndroidBigPictureStyle,
 ): AndroidBigPictureStyle {
-  if (!isString(style.picture) || !style.picture) {
+  if (
+    (!isString(style.picture) && !isNumber(style.picture) && !isObject(style.picture)) ||
+    !style.picture
```

Comment:
```suggestion
    !style.picture
```

Image assets numbers start from `0` afaik - so I think this validation fail in that scenario, might be best to remove this check from here, and also from the other places that have done it - instead expand the check to check for string length if it's a string. (isObject already handles the undefined scenario)